### PR TITLE
DAPI: display a placehoder when there is no key indicator yet

### PR DESCRIPTION
**Pull request by rouliane**
**Description (for Contributor and Core Developer)**
image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const EmptyKeyIndicators = () => {
+  const translate = useTranslate();
+
+  return (
+    <Container>
+      <img src="bundles/akeneodataqualityinsights/images/empty-key-indicators.svg" />
+      <p>{translate('akeneo_data_quality_insights.dqi_dashboard.key_indicators.no_data')}</p>
+      <p dangerouslySetInnerHTML={{
+        __html: translate('akeneo_data_quality_insights.dqi_dashboard.key_indicators.no_data_subtitle', {link: 'https://help.akeneo.com'})
```
Comment (rouliane) [inline src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx:25]: waiting for the real link

**Review by LaurentPetard — APPROVED** [review]

File: src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx (hunk 1 above)
Comment (rouliane) [reply src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx]: discussed with Lucas: we don't show the link.

File: src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import styled from "styled-components";
+import {useTranslate} from "@akeneo-pim-community/legacy-bridge";
+
+const Container = styled.div`
+  text-align: center;
+  font-size: ${({theme}) => theme.fontSize.big};
+  margin: 10px 0 0 0;
+  width: 100%;
+  
+  a {
```
Comment (olivierpontier) [inline src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx:11]: this style rule for links seems to not be used. Could we remove it?

**Review by olivierpontier — APPROVED** [review]

**Review by AymericPlanche — APPROVED** [review]

File: src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx (hunk 2 above)
Comment (rouliane) [reply src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/EmptyKeyIndicators.tsx]: good catch, sorry about that

**Pull request merged**